70_Pushover.pm

```diff
@@ -1228,19 +1258,18 @@ sub Pushover_SetMessage2 ($$$$) {
 
         if (
             !$callback
-            || (   $values{action} !~ /^http[s]?:\/\/.*$/
-                && $values{action} =~ /^[\w-]+:\/\/.*$/ )
+            || $values{action} =~ /^[\w-]+:\/\/.*$/
           )
         {
             $url = $values{action};
-            $values{expire} = undef;
+            if ( $values{priority} < 2 ) {
+                $values{expire} = undef;
+            }
         }
         else {
             $url =
                 $callback
-              . "?acknowledged=1&acknowledged_by="
-              . $hash->{USER_KEY}
-              . "&FhemCallbackId="
+              . "?FhemCallbackId="
```

Comment:
7) Removal of checking if the user key is the key configured in the pushover device in FHEM, since this would fail for group keys